### PR TITLE
#185172078 configure react router

### PR DESCRIPTION
**Pull request by Edcreation**
#### Description of Task to be completed?
- Setup react router  (React-router-dom)
#### How should this be manually tested?
- By Reviewing codes you can identify react router being used
#### Any background context you want to provide?
- Set Up Agreed upon folder structure
#### What are the relevant pivotal tracker/Trello stories?
- 185172078

### CODE REVIEW

**Review by Pacific911 — APPROVED** [review]

**Review by ishimwe5555 — APPROVED** [review]
LG

**Review by JosephNgabo — APPROVED** [review]

File: jest.config.cjs [hunk 1]
```diff
@@ -17,7 +17,7 @@ module.exports = {
     '\\.module.scss$',
     '\\.module.css$',
   ],
-  testMatch: ['<rootDir>/src/tests/**/*.test.(ts|tsx)', '<rootDir>/src/tests/**/*.spec.(ts|tsx)'],
+  testMatch: ['<rootDir>/src/**/*.test.(ts|tsx)', '<rootDir>/src/**/*.spec.(ts|tsx)'],
```
Comment (munezerobagira) [inline jest.config.cjs:20]: can you add collectCoverageFrom also

File: package.json [hunk 2]
```diff
@@ -43,23 +45,23 @@
...
+    "husky": "^8.0.3",
     "identity-obj-proxy": "^3.0.0",
     "lint-staged": "^13.2.2",
     "prettier": "^2.8.8",
     "typescript": "^5.0.2",
     "vite": "^4.3.2"
   },
+  "main": "index.js",
+  "repository": "https://github.com/atlp-rwanda/Team-Sostene-E-commerce-fe.git",
+  "author": "AdrineUwera <a.uwera@alustudent.com>",
```
Comment (munezerobagira) [inline package.json:57]: why did you add this line

**Review by munezerobagira — CHANGES_REQUESTED** [review]

File: jest.config.cjs (hunk 1 above)
Comment (Edcreation) [reply jest.config.cjs:20]: Added!

**Review by munezerobagira — APPROVED** [review]

**Pull request merged**